apps/explorer/src/pages/address-result/AddressResult.tsx

```diff
@@ -138,9 +139,15 @@ function SuiNSAddressResult({ name }: { name: string }) {
 }
 
 export default function AddressResultPage() {
+	const isObjectAddressUnifyPageEnabled = useFeatureIsOn('object-address-unify-page');
```

Comment:
I don't think there's a lot of value in adding this feature flag, we instead can just flip this on by adding the router-level redirects.

Reply:
Removed the flag